react/modules/deprecated.tsx

```diff
@@ -14,19 +14,12 @@
  * */
 import React, { Component, ComponentType } from 'react'
 
-interface UseNewProps {
-  submitFilterLable: string
-  newFilterLable: string
-}
-
-interface UseNewComponent {
-  old: string
-  new: string
-}
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type genericObj = Record<string, any>
```

Comment:
⛏️ This use of `Record` with any is a bit strange to me, I would go for: 
```suggestion
interface genericObj {
    [key: string]: any
}
```
I don't know if this works the same as the line above. What do you think?

Reply:
it is exactly the same thing, for those who are used to typescript Record it is much simpler to read. First parameter is the key and the second is the value, always.